kv/tlsconfig.go

```
@@ -0,0 +1,42 @@
+package kv
+
+import (
+	"crypto/tls"
+	"crypto/x509"
```

golangci-lint

File is not `goimports`-ed

(goimports)